packages/terra-notification-dialog/src/ContentLayoutAsList.module.scss

```diff
@@ -0,0 +1,22 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './clinical-lowlight-theme/ContentLayoutAsList.module';
+@import './orion-fusion-theme/ContentLayoutAsList.module';
+
+:local {
+  ul {
```

Comment:
should we be using the classes `list` and `list-item` instead of the element tag? if this stylesheet is imported, wouldn't all ul and li tags get these styles on the page, regardless of the class being locally scoped?

Reply:
....yeah... 😅

Reply:
fixed: https://github.com/cerner/terra-framework/pull/1105/commits/022584400b4aaafcf4ad53ff1ed2e2b2eaec08a5#diff-92826ca07f400310e346d2a5989c48deR8

can't believe I did that... 🤦‍♀️